src/main/java/org/jitsi/jicofo/recording/jibri/JibriRecorder.java

```diff
@@ -323,31 +340,34 @@ private void processJibriIqFromMeet(JibriIq iq, XmppChatMember sender)
             if (jibriJid == null)
             {
                 sendErrorResponse(
-                    iq, XMPPError.Condition.service_unavailable, null);
+                        iq, XMPPError.Condition.service_unavailable, null);
                 return;
             }
 
-            final String roomName = MucUtil.extractName(senderMucJid);
-
-            try
+            XMPPError err = null;
+            for (int i = 0; i < NUM_RETRIES; i++)
```

Comment:
Also I am a bit worried about blocking packet processor thread on this loop. It will be called either from onJibriOffline or processJibriIqFromMeet where both are triggered by Smack's PacketListener. Currently timeout for packets sent in smack is set to 15 seconds which means that the thread can be blocked for up to 45 seconds if we retry 3 times. This will block packet processing in all other parts of Jicofo.